From 839a1935c5433fe973008f231aeb247df0344eca Mon Sep 17 00:00:00 2001
From: Lukas Nykryn <lnykryn@redhat.com>
Date: Tue, 9 Jan 2018 12:59:19 +0100
Subject: [PATCH] shared/cgroup-utils: _CGROUP_CONTROLLER_MASK_ALL does not
 cover CGROUP_PIDS

7d44d0d43465892d4753ff50592588f49d56cf95 added a CGROUP_PIDS but
did not bump _CGROUP_CONTROLLER_MASK_ALL.

RHEL-only
Resolves: #1532586
---
 src/shared/cgroup-util.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/shared/cgroup-util.h b/src/shared/cgroup-util.h
index 31bd8d311..e76cd334d 100644
--- a/src/shared/cgroup-util.h
+++ b/src/shared/cgroup-util.h
@@ -36,7 +36,7 @@ typedef enum CGroupControllerMask {
         CGROUP_MEMORY = 8,
         CGROUP_DEVICE = 16,
         CGROUP_PIDS = 32,
-        _CGROUP_CONTROLLER_MASK_ALL = 31
+        _CGROUP_CONTROLLER_MASK_ALL = 63
 } CGroupControllerMask;
 
 /*
